src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -136,12 +136,22 @@ public void insertWhenValidDataReturnsTrue() throws SQLException {
     }
 
     @Test
-    public void insertWhenInvalidTableNameReturnsFalse() throws SQLException {
+    public void insertWhenTypeMismatchParameterReturnsFalse() throws SQLException {
+        databaseManager.connect(TEST_DB_NAME, DB_USER, DB_USER_PASSWORD);
```

Comment:
Why you use connect in every method instead of @Before?

Reply:
There's a tests of a method connect() where shouldn't be a preliminary connection instantiating